dask/_collections.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import annotations
```

Comment:
I had to remove this from dask-expr to avoid circular imports, that's a change that's not contained to array-expo